server/views/components/image/index.njk

```diff
@@ -0,0 +1,38 @@
+<figure class="figure
+               {{ ' figure--full' if full }}
```

Comment:
It does annoyingly, and is fine.
Using the `{{- blahblah -}}` syntax seems to screw up fractal.

I'll file a bug report.